docs/Documentation/Drivers/Classic/FrozenSoilOutputFile.md

```diff
@@ -2,7 +2,7 @@
 
 When the model is run with the frozen soil algorithms, a third output file is produced which contains soil thermal output parameters. Since the frozen soils algorithm only works in with the full energy balance, there is only one format for the output file.
```

Comment:
Yes, `FULL_ENERGY` simply controls whether VIC iterates to find the surface temperature that balances the surface energy budget.  Even when `FULL_ENERGY